lib/active_admin/form_builder.rb

```diff
@@ -11,6 +11,16 @@ def input_wrapping(&block)
   end
 end
 
+module Formtastic
+  class FormBuilder
+    # Prevent sensitive fields from being shown in forms by default
+    # NOTE: Formastic::Helpers::InputsHelper.inputs calls
+    #   ``Formtastic::FormBuilder.skipped_columns`` explicitly so we need to
+    #   monkeypatch instead of use inheritance
+    self.skipped_columns += ActiveAdmin.application.filter_attributes
+  end
+end
```

Comment:
Do you think this is something that we should fix (or at least report to) formtastic? Also, could you add a spec for the monkeypatch?

Reply:
I've opened a [PR for formtastic](https://github.com/justinfrench/formtastic/pull/1273). Not sure wether there is much support for subclassing from FormBuilder from their side because they explicitly call Formtastic::FormBuilder in various other places, too.